src/project/internal/exportprojectscenario.cpp

```diff
@@ -56,6 +56,13 @@ bool ExportProjectScenario::exportScores(const INotationPtrList& notations, cons
         return false;
     }
 
+    // Parts that are closed may have never been laid out, so we lay them out now
+    for (INotationPtr notation : notations) {
```

Comment:
I would recommend moving this code to MasterNotation::initExcerpts

Reply:
done, thanks!